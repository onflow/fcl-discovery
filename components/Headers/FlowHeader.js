import { Center, Flex, Image, Text } from '@chakra-ui/react'

export default function FlowHeader() {
  return (
    <>
      <Center>
        <Flex flexDirection="column" alignItems="center" marginBottom={4}>
          <Image 
            src="/logo.svg" alt="Logo"
            w='140px'
            h='auto'
            marginBottom={3}
          />
          <Text color='grey'>
            Choose a Provider
          </Text>
        </Flex>
      </Center>
    </>
  )
}
