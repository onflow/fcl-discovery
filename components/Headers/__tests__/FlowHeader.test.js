import { render } from '@testing-library/react'
import FlowHeader from '../FlowHeader'

describe('Component: FlowHeader', () => {
  test('should render the FlowHeader component', () => {
    const { container } = render(<FlowHeader />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
