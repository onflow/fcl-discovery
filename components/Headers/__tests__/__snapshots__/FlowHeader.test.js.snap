// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: FlowHeader should render the FlowHeader component 1`] = `
<div
  class="css-gmuwbf"
>
  <div
    class="css-1j99xrl"
  >
    <img
      alt="Logo"
      class="chakra-image css-kl8a02"
      src="/logo.svg"
    />
    <p
      class="chakra-text css-mwent8"
    >
      Choose a Provider
    </p>
  </div>
</div>
`;
